frontend/pages/policies/edit/components/PolicyAutomations/PolicyAutomations.tsx

```
@@ -41,7 +41,7 @@ const PolicyAutomations = ({
   const hasPatchSoftware = !!storedPolicy.patch_software;
   const hasSoftwareAutomation = !!storedPolicy.install_software;
   const showCtaCard =
-    isPatchPolicy && hasPatchSoftware && !hasSoftwareAutomation;
+    isPatchPolicy && hasPatchSoftware && !hasSoftwareAutomation && onAddAutomation;
```

ESLint

[error] Replace `·hasPatchSoftware·&&·!hasSoftwareAutomation·&&` with `⏎····hasPatchSoftware·&&⏎····!hasSoftwareAutomation·&&⏎···`

(prettier/prettier)